### PR TITLE
Increase time range for PrometheusHAGroupCrashlooping alert

### PR DESCRIPTION
**Pull request by nikosmeds**
Increase the PrometheusHAGroupCrashlooping alert time range to 1 hour, as 30 minutes does not catch all crashlooping systems.

Signed-off-by: Niko Smeds <nikosmeds@gmail.com>

### CODE REVIEW

**Comment by beorn7** [comment]
There is actually a trade-off here: You might very well have a good number of clean restarts if you are pushing new command line flags or are trying out different upgrades and downgrades. Luckily, we have the metrics for clean restarts now.

How about increasing the time only for the unclean restarts? I would be very careful with raising it for the clean restarts as that's prone to false positives.

File: documentation/prometheus-mixin/alerts.libsonnet [hunk 1]
```diff
@@ -418,7 +418,7 @@
             },
             annotations: {
               summary: 'More than half of the Prometheus instances within the same HA group are crashlooping.',
-              description: '{{ $value | humanizePercentage }} of Prometheus instances within the %(prometheusHAGroupName)s HA group have had at least 5 total restarts or 2 unclean restarts in the last 30m.' % $._config,
+              description: '{{ $value | humanizePercentage }} of Prometheus instances within the %(prometheusHAGroupName)s HA group have had at least 5 recent total restarts or 2 recent unclean restarts.' % $._config,
```
Comment (nikosmeds) [inline documentation/prometheus-mixin/alerts.libsonnet:421]: Opted for a shorter description. If we'd prefer to keep time ranges included in the alert's description:

```suggestion
              description: '{{ $value | humanizePercentage }} of Prometheus instances within the %(prometheusHAGroupName)s HA group have had at least 5 total restarts in the last 30m or 2 unclean restarts in the last 1h.' % $._config,
```

Comment (beorn7) [reply documentation/prometheus-mixin/alerts.libsonnet:421]: I actually like the description with the time ranges more. It's only barely longer but much more to the point.

**Review by beorn7 — COMMENTED** [review]
Otherwise, I think this should be fine. I'll wait a bit more to see if someone else has objections.

**Review by beorn7 — APPROVED** [review]

**Pull request merged**